library/cc/engine_builder.h

```diff
@@ -38,6 +38,7 @@ class EngineBuilder {
   EngineBuilder& setAppId(const std::string& app_id);
   EngineBuilder& setDeviceOs(const std::string& app_id);
   EngineBuilder& setStreamIdleTimeoutSeconds(int stream_idle_timeout_seconds);
+  EngineBuilder& setPerTryIdleTimeoutSeconds(int per_try_idle_timeout_seconds);
```

Comment:
our Kotlin and Swift engines use `add...` naming but the c++ seems to prefer `set...` one... I guess it's fine to keep using `set` in here (which I personally prefer)